react-form-controls/form-controls/src/RegistrationFormControlled.tsx

```diff
@@ -0,0 +1,34 @@
+import { useState } from 'react';
+
+export default function RegistrationFormcontrolled() {
+  const [username, isUsername] = useState('');
```

Comment:
Use `set` as the prefix on your setter function names, instead of `is`.

`set` is a verb that implies action. `is` suggests that the variables are booleans.